src/ui-test/baseExtensionTest.ts

```diff
@@ -0,0 +1,40 @@
+import { ExtensionsViewSection, ActivityBar, ExtensionsViewItem, QuickOpenBox } from 'vscode-extension-tester';
+import { ProjectInitializer } from './common/projectInitializerConstants';
+import { openCommandPrompt } from './common/commonUtils';
+import { expect } from 'chai';
+
+export function baseExtensionUITest() {
+    describe('Verify extension\'s base assets available after install', () => {
+
+        let inputBox: QuickOpenBox;
+
+        it('Command Palette prompt knows project initializer commands', async function () {
+            this.timeout(4000);
```

Comment:
Can you try to use a conditional wait?
Have you spotted something missing to use ones?

Reply:
This is the test method timeout.

Reply:
ah ok, my bad.

Reply:
I cannot closed it as "resolved", missing rights I guess. (and also for all the other similar comments :-( )